modules/oe_theme_content_tender/oe_theme_content_tender.install

```diff
@@ -19,19 +20,25 @@ function oe_theme_content_tender_install() {
   }
 
   // Override entity view displays.
-  $config_name = 'core.entity_view_display.node.oe_tender.default';
   $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_tender') . '/config/overrides');
-  $config_values = $storage->read($config_name);
-
-  // Load existing configuration.
-  $config_manager = \Drupal::service('config.manager');
-  $entity_type = $config_manager->getEntityTypeIdByName($config_name);
-  $config_entity_storage = $config_manager
-    ->getEntityTypeManager()
-    ->getStorage($entity_type);
-  $config_entity = $config_entity_storage->load($config_values['id']);
-
-  // Update configuration.
-  $updated_config_entity = $config_entity_storage->updateFromStorageRecord($config_entity, $config_values);
-  $updated_config_entity->save();
+  $displays = [
```

Comment:
There were discussion how better to update configuration and it was decided to use code as above. You can use loop to update view displays.
See example https://github.com/openeuropa/oe_content/blob/master/modules/oe_content_entity/modules/oe_content_entity_contact/oe_content_entity_contact.post_update.php#L46.

Reply:
@ademarco and @nagyad what way of update will we use? You both provide different requirements to it.

Reply:
The linked case is different than this one, this one is a hook_install() while the one in @yenyasinn 's comment is a hook_post_update_X(). Still it would be good to pick one way of doing this in hook_install() too.

I've checked and all the hook_install() are actually consisten with what's proposed here (i.e. using `EntityViewDisplay`) apart from `oe_theme_content_entity_organisation_install()`.

@yenyasinn I wonder if `updateFromStorageRecord()` created the entity if not existing, I doubt that, but we could try it on a separate branch. on `oe_theme_content_entity_organisation_install()` we used that since the `core.entity_view_display.oe_organisation.oe_stakeholder.default` is created already by the  `oe_content_entity_organisation` module while, in this case, we create / update `default` and `teaser` and `teaser` does/might not exists, while default `surely` does.